### ISSUE TITLE
Help to use

### ISSUE THREAD

**Issue by andrecordeiro** [TeamFreeHugs/Whatsapp-Chatbot#2]
@eyeballcode Hello, I do not quite understand how to use this chatbot. I started the web server and it is running on port 3000. I opened the url https://web.whatsapp.com. And now what do I do?

This is my console: 
![capturar](https://cloud.githubusercontent.com/assets/7633970/13919838/89c95a28-ef4e-11e5-8e96-8969a11a92bc.PNG)

**Comment by eyeballcode** [comment]
So on whatsapp web, you have a QR code:
![screenshot from 2016-03-21 23 09 15](https://cloud.githubusercontent.com/assets/6756003/13923075/131e35c6-efba-11e5-8036-5c577f09a43a.png)

Open the QR code scanner on the phone and scan it:
![help_chatbot_ws](https://cloud.githubusercontent.com/assets/6756003/13923261/cf3ef0f6-efba-11e5-960c-e08b798aed76.png)

Give it a moment, it should change the page in the browser. Click on a chat and right click, click inspect element then go to the JS console

I've found that right clicking on the chat info bar works.

Then paste in the code from `WhatsappUtils.js` inside. If it blocks a popup, allow it, then run the code again.

And you can play with this starter code:

```
window.whatsapp = initWhatsappAPI();

whatsapp.attachHandler({
    newMessage: function (text) {
        console.log("NEW MESSAGE: " + text);
    }
});
whatsapp.startWatching();
```

Also, note that I am working on a pure nodejs way of doing this. No more messy CORS getarounds.

**Comment by andrecordeiro** [comment]
You can help me via teamviewer ?

**Comment by eyeballcode** [comment]
What is the problem with the chatbot?

**Comment by andrecordeiro** [comment]
I opened the inspector, but did not understand what the code should necklace and where necklace.

**Comment by eyeballcode** [comment]
Necklace? What necklace?

**Comment by andrecordeiro** [comment]
when you say : Then paste in the code from WhatsappUtils.js inside . If it blocks a popup, allow it , Then run the code again . I did not understand.
![capturar](https://cloud.githubusercontent.com/assets/7633970/13925174/21aaeaf2-ef66-11e5-8dc5-b500f58d8b72.PNG)

**Comment by eyeballcode** [comment]
If the server is started then load [this](http://localhost:3000/javascript/WhatsappUtils.js) link to see the file

**Comment by andrecordeiro** [comment]
Ready friend, opened the url . Now what should I do ?
![capturar](https://cloud.githubusercontent.com/assets/7633970/13926057/6886f206-ef69-11e5-9ce5-a194c6696234.PNG)

**Comment by andrecordeiro** [comment]
I need to create a script in node.js and put this code in and run ?

window.whatsapp = initWhatsappAPI();

whatsapp.attachHandler({
    newMessage: function (text) {
        console.log("NEW MESSAGE: " + text);
    }
});
whatsapp.startWatching();

**Comment by andrecordeiro** [comment]
was it? ![capturar](https://cloud.githubusercontent.com/assets/7633970/13926522/7a792374-ef6b-11e5-8792-30cdb33b7b42.PNG)

**Comment by andrecordeiro** [comment]
when I open the http://localhost:3000/WhatsappUtils.js, an error occurs. You know how I can solve?

![capturar](https://cloud.githubusercontent.com/assets/7633970/13928332/fa0e8162-ef73-11e5-8e02-eaf3f5a04518.PNG)

**Comment by andrecordeiro** [comment]
I could make it work. thank you so much. how do I let the chatbot working for several people ? It is only working for just a conversation , if I make a chat with another person, for the system and have to start again.

**Comment by eyeballcode** [comment]
Yep. You could try a `MutationObserver` on the chat list, but right now I can't really get it to work

**Comment by sanjayradadiya** [comment]
i am starting to implement MutationObserver but now a day i can't got success.

**Comment by andrecordeiro** [comment]
good morning my friend. very good at your job. It was very good indeed.

**Comment by eyeballcode** [comment]
I might be able to get a basic version of the pure nodejs api soon, and it should have multi chats at once.

**Comment by sanjayradadiya** [comment]
can you give me basic idea on which you are working on

**Comment by eyeballcode** [comment]
It is a new version of the API that uses jsdom to do all the magic instead of a browser.
This lets it do all the DOM magic inside nodejs, without a browser. This also allows me to upload files (hopefully) and be easier to setup.

**Comment by sanjayradadiya** [comment]
you mean this is also multi user chat enabled logic using node.js

**Comment by andrecordeiro** [comment]
Hello Friend. I would like to do something like this.

https://m.youtube.com/watch?v=0JqQqL0-Nc8

**Comment by eyeballcode** [comment]
Not yet, but when I can get the nodejs version to work

**Comment by andrecordeiro** [comment]
Fantastic!

**Comment by andrecordeiro** [comment]
How can we talk in private?

**Comment by eyeballcode** [comment]
I don't, but we could try gitter?

Get an account on gitter, then go [here](https://gitter.im/eyeballcode/Room-for-eyeballcode-and-andrecordeiro)

**Comment by andrecordeiro** [comment]
Gitter

404. This is not the chat you're looking for. Sorry. :(

**Comment by eyeballcode** [comment]
Signup first. Then I can add you to the room

**Comment by andrecordeiro** [comment]
https://gitter.im/info2biz?utm_source=share-link&utm_medium=link&utm_campaign=share-link

**Comment by sanjayradadiya** [comment]
dear all 
can we have any updates regarding dom majic inside node.js

**Comment by andrecordeiro** [comment]
Hello Friend. you released the new version ? I did not understand your message.

**Comment by sanjayradadiya** [comment]
no sir i have't any update i just asking Que that any update occurred in this script or not

**Comment by andrecordeiro** [comment]
Hello . okay?

**Comment by eyeballcode** [comment]
Currently, I don't have an update as I can't get jsdom to work as I want it to.

**Comment by andrecordeiro** [comment]
good evening. my friend, you could pass me how to operate the system without having to leave the browser open ? Then we solve the rest . Can you explain me?
or provide what you have done for me to take a look and try to help.

**Comment by johnnogent** [comment]
Hello eyeballcode, 

I have followed the readme, paste both files in JS console, and nothing happens apart another browser fully blank. What should be the outcome?

Thanks

**Comment by eyeballcode** [comment]
Hi, this repository is outdated, please use
https://github.com/TeamFreeHugs/WhatsApp-Chatbot-2.0 instead. Thanks
